content/kubermatic/master/requirements/cloud_provider/kubevirt.en.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "KubeVirt"
+date = 2021-02-01T14:46:15+02:00
+weight = 7
+
++++
+
+## KubeVirt
+Once KubeVirt is installed as what the [official documentation](https://kubevirt.io/quickstart_cloud/) guides, a few
+steps should be followed in order to use KubeVirt with KKP.
+
+### StorageClass Requirements 
+KKP uses [Containerized Data Importer](https://github.com/kubevirt/containerized-data-importer) (CDI) to import images and 
+provision volumes to launch the VMs. CDI provides the ability to populate PVCs with VM images or other data upon creation. 
+The data can come from different sources: a URL, a container registry, another PVC (clone), or an upload from a client.
+For more information about the requirements of Kubernetes in general and CDI in specific, please follow the documentation
+for PV,PVC and DV [here](https://github.com/kubevirt/containerized-data-importer/blob/master/doc/basic_pv_pvc_dv.md).
+
+### KubeVirt Operator and Containerized Data Importer Version
+KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >= v1.19.0. There are no hard requirements 
+to run KubeVirt, however a kubernetes cluster consists of 3 nodes with 2 CPUs, 4GB of RAM and 30GB of storage, to have a 
+minimal installation.
+
+### Use KKP with KubeVirt 
+In order to allow KKP to provision VMs(worker nodes) in KubeVirt, users provide the kubeconfig of the kubernetes cluster 
+where KubeVirt installation is running. Users can add the content of the kubeconfig file in the third step of the cluster
```

Comment:
"where _the_ KubeVirt installation is running" or "where KubeVirt is running"

Reply:
Probably should be where the KubeVirt cluster is running.